targets/notion/index.js

```diff
@@ -0,0 +1,24 @@
+const slack = require('../../targets/slack')
+const { notion } = require('../../sources/notion/api')
+const { mdNotionConverter } = require('./converter')
+
+
+const { DATABASE_ID = 'adf0c7124e1e44ff851e254dbe36015c' } = process.env
+const today = `${new Date().toISOString().split('T')[0]}`
+
+module.exports.uploadMD = async (post, tag) => {
+  const page = await notion.pages.create({
+    parent: { type: 'database_id', database_id: DATABASE_ID },
+    properties: {
+      Digest: {
+        title: [{
+          text: { content: post.title },
+        }],
+      },
+      Date: { type: 'date', date: { start: today } },
+      Tags: { multi_select: [{ name: tag }] },
+    },
+    children: mdNotionConverter(post.content),
+  })
+  slack.notionNotify({ url: page.url, title: post.title, summary: post.summary })
```

Comment:
should we either internalize a control flag for the `uploadMD` function in the params, such as

```js
module.exports.uploadMD = async (post, tag, { slack = true } = {}) => {
  ...
  if (slack) {
    slack.notionNotify(...)
  }
}
```

or externalize the call after `notionTarget.uploadMD` within each of the CLI function handlers